examples/scale-to-zero-with-keda/README.md

```diff
@@ -12,6 +12,15 @@ It realises the following scenario:
 
 ![scenario](assets/scaling-scenario.png "Scenario")
 
+1. `http request` arrives in the `http-proxy-fn` Pod
+2. `http-proxy-fn` sends a message to the initially scaled-down `scalable-worker-fn` using Kyma Eventing
+3. Eventing keeps resending the message until it's received and acknowledged
+4. Prometheus records the traffic between Eventing and the `scalable-worker-fn` Kubernetes service
+5. KEDA reads a non-zero request rate targeted for the `scalable-worker-fn` Kubernetes service
+6. KEDA scales up the `scalable-worker-fn` Pod
+7. `scalable-worker-fn` eventually receives the message from Eventing and processes it; Eventing receives the acknowledgment and stops resending 
```

Comment:
this number should be marked in the diagram on the arrow between k8s service and pods